.github/workflows/release.yml

```diff
@@ -61,8 +61,8 @@ jobs:
     - name: Publish to OSSRH
       env:
         GITHUB_DEPENDENCY_GRAPH_REF: refs/heads/main
-        ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.ORG_OSSRH_USERNAME }}
-        ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.ORG_OSSRH_PASSWORD }}
+        ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.CENTRAL_SONATYPE_TOKEN_USERNAME }}
```

Comment:
While at it, I propose to align with ORT and call these `MAVEN_CENTRAL_USERNAME` and `MAVEN_CENTRAL_PASSWORD`.

Reply:
The names are provided by EF, I cannot change them.